### PR TITLE
chore(stoneintg-137) Replace docker with podman

### PR DESCRIPTION
**Pull request by 14rcole**

### CODE REVIEW

**Review by jsztuka — APPROVED** [review]
changes lgtm, let me know if you managed to test them, we can figure something out

File: .github/workflows/clair-in-ci.db.yaml
Comment (14rcole) [inline .github/workflows/clair-in-ci.db.yaml:32]: This is the one piece of this I haven't been able to test.  This action is hosted at github.com/redhat-actions/podman-login.  Will github just automatically clone the repo to use the action or are there some additional steps I need to take to make sure this works properly?

Comment (MartinBasti) [reply .github/workflows/clair-in-ci.db.yaml:32]: I'd just trust it

File: .github/workflows/clair-in-ci.db.yaml
Comment (MartinBasti) [inline .github/workflows/clair-in-ci.db.yaml:39]: How did you installed podman here?

Comment (MartinBasti) [reply .github/workflows/clair-in-ci.db.yaml:39]: okay, I don't see it being explicitly installed even here https://github.com/redhat-appstudio/hacbs-test/blob/main/.github/workflows/clam-db.yaml

File: .github/workflows/clair-in-ci.db.yaml
Comment (MartinBasti) [reply .github/workflows/clair-in-ci.db.yaml:52]: https://github.com/redhat-appstudio/clair-in-ci-db/pull/6 I wrote PR to use tags in the exactly the same way as ClamAV, it's just failing I don't know why

Comment (14rcole) [reply .github/workflows/clair-in-ci.db.yaml:52]: Do you mean that we should get rid of the `previous` tag in order to align with ClamAV?  Doesn't keeping the tag give us some redundancy in case we need to rollback?

Comment (MartinBasti) [reply .github/workflows/clair-in-ci.db.yaml:52]: calculated timestamp is unique tag, that won't change, we will use it for rollbacks if needed.
Tags latest and v1 are floating and updated after testing, and we should push to both, latest for bw compatibility.

Comment (14rcole) [reply .github/workflows/clair-in-ci.db.yaml:52]: Ack, I'll wait for #6 to be merged then update the tagging here accordingly

File: .github/workflows/clair-in-ci.db.yaml
Comment (14rcole) [inline .github/workflows/clair-in-ci.db.yaml:49]: Technically `podman pull` isn't necessary - `podman run` will pull the image if it doesn't exist locally.  However, should it be kept for clarity?

File: .github/workflows/clair-in-ci.db.yaml
Comment (MartinBasti) [inline .github/workflows/clair-in-ci.db.yaml:64]: is skopeo installed there?

Add --all param to skopeo, so it copies everything including manifests lists (if exists, but eventually they will exist)

Also we should push to `v1` tag as with clamav and slowly deprecate usage of latest
```suggestion
          skopeo copy --all ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.TO_TEST }} ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.LATEST_TAG }}
                    skopeo copy --all ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.TO_TEST }} ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.MAJOR_VERSION_TAG }}
```

File: .github/workflows/clair-in-ci.db.yaml
Comment (MartinBasti) [inline .github/workflows/clair-in-ci.db.yaml:40]: I'm not sure what's the purpose of this for docker, and I'm really not sure if this is needed for podman, mainly we don't build here anything anymore

Comment (14rcole) [reply .github/workflows/clair-in-ci.db.yaml:40]: I'm not sure either, I kept it in to be safe.  I can get rid of it.

**Review by MartinBasti — APPROVED** [review]

**Pull request merged**
